src/main/resources/config.yml

```diff
@@ -11,4 +11,9 @@ contributors:
   - "<Notch><Creator of Minecraft><563.5, 105, 554.5>"
   - "<stxphen><Event Host><559.5, 105, 554.5>"
 
+# Donation Auth
+donations:
+  authToken: ""
```

Comment:
how long is the token valid for?

Reply:
Now that you mention it, only 2 hours, should be a simple fix tho.

Reply:
Once you have that fix implemented, please provide a `.yml` comment above this line stating its general expiry.